jupyterhub_singleuser_profiles/profiles.py

```diff
@@ -214,6 +214,30 @@ def env_dict_to_list(self, env_data):
 
     return result
 
+  @classmethod
+  def get_image_list(self, imagestream_list, last_image):
```

Comment:
Please rename this to `get_image_list_form` so that it is consistent with other methods producing form sections